Thinktecture.Relay.Server/Config/Configuration.cs

```diff
@@ -134,6 +136,19 @@ public Configuration(ILogger logger)
 			OAuthSharedSecret = ConfigurationManager.AppSettings["OAuthSharedSecret"];
 			OAuthCertificate = ConfigurationManager.AppSettings["OAuthCertificate"];
 
+			if (!TimeSpan.TryParse(ConfigurationManager.AppSettings["HstsHeaderMaxAge"], out tmpTimeSpan))
+			{
+				tmpTimeSpan = TimeSpan.FromDays(365);
+			}
+
+			HstsHeaderMaxAge = tmpTimeSpan;
+
+			HstsIncludeSubdomains = false;
```

Comment:
This can be set by negating the check in the next line and using `tmpBool` like the other items before.